OpenSim/Common/ComponentConnector.h

```diff
@@ -446,17 +491,9 @@ class OSIMCOMMON_API AbstractInput : public AbstractConnector {
             channelName = "";
         }
         
-        // Annotation.
-        if (leftParen != std::string::npos && rightParen != std::string::npos) {
-            annotation = connecteeName.substr(leftParen + 1,
-                                              rightParen - (leftParen + 1));
-        } else {
-            if (!channelName.empty()) {
-                annotation = channelName;
-            } else {
-                annotation = outputName;
-            }
-        }
+        // Alias.
+        alias = connecteeName.substr(leftParen + 1, rightParen - (leftParen + 1));
```

Comment:
> The checks for npos are already there.

Previously, there were checks for `npos` for the parentheses:

```
if (leftParen != std::string::npos && rightParen != std::string::npos) {
```

These have been removed.

The only check for `npos` I see now is

```
if (colon != std::string::npos) {
```

What are you referring to?